tests/deprecated_classes.py

```diff
@@ -405,7 +405,7 @@
         'airflow.contrib.hooks.aws_hook.AwsHook',
     ),
     (
-        'airflow.providers.amazon.aws.hooks.aws_dynamodb.AwsDynamoDBHook',
+        'airflow.providers.amazon.aws.hooks.dynamodb.AwsDynamoDBHook',
```

Comment:
So we don't have a deprecation mechanism between backport package versions, right? Only between Airflow 1.10.x and latest backport packages.

Reply:
We can create a followup task for that